modules/op-tkn-clustertask-management.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-clustertask-management-commands_{context}"]
+= ClusterTask management commands
+
+== clustertask
+Manage ClusterTasks.
+
+.Example: Display help
```

Comment:
For all of the examples, I would reccomend following a similar structure to the odo CLI reference assembly (https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_developer_cli/odo-cli-reference.html) 

For example: s/Example: Display help/Example display help

Reply:
Hi @bmcelvee ,
We have followed the oc Developer CLI commands format in tkn:
https://docs.openshift.com/container-platform/4.3/cli_reference/openshift_cli/developer-cli-commands.html

The reason being:
1. Structure of the oc developer CLI is similar to tkn. 
2. Following the oc developer CLI structure makes the doc more expandable in future, in case we need to highlight something for a specific command. On the other hand,  in odo all the examples are clubbed together as a code block, leaving little room for expansion ( eg: highlighting a specific parameter, or adding a note for a command)

Please do let me know your thoughts. 
Thanks!!

Reply:
Makes sense, we can leave it as. Thank you for the explanation!